dandiapi/api/manifests.py

```diff
@@ -47,13 +47,6 @@ def s3_url(path: str):
 
 def manifest_location(version: Version):
     """Calculate the manifestLocation field for a Version."""
-    if version.version == 'draft':
-        return [
-            (
-                f'https://api.dandiarchive.org/api/dandisets/{version.dandiset.identifier}'
-                f'/versions/draft/assets/'
-            )
-        ]
```

Comment:
What happens if you try to get manifest for a draft dandiset whose first 1am yamling hasn't happened yet?

Reply:
also is there a reason not keep this path for this function (since this is what is injected into the landing page metadata), since this will always reflect current state of db, whereas the s3 url will reflect our periodic update.

Reply:
also the s3 location will always have the manifest written out as assets.yaml/jsonld, hence changing this function i don't think is necessary.

Reply:
I recalled this being a hack to work around the fact that we didn't have draft assets. If it's actually the preferred value, I will just revert this change.